### ISSUE TITLE
Android Auto support

### ISSUE THREAD

**Issue by tcressy** [AndBible/and-bible#2905]
I use Android Auto in my car all of the time. Recently, a friend connected their iPhone (Apple Car Play) in my car, as I was surprised to see the MySword Bible app come up in Car Play. When selecting it, we were presented with a list of the bible books, then once a book was selected, a chapter list. Clicking on a chapter, the KJV audio started playing for that chapter.

I thought, what a great idea. Wouldn't it be amazing if And Bible supported Android Auto and allowed us to select and play Bible passages in our car!

Please can you consider implementing this! (If you did, you would be one of the very few Bible apps that would support this, making yours stand out!)

Many thanks for your consideration,
Tim

**Comment by tuomas2** [comment]
Good idea, but I just don't happen to have a fancy car with Android in it :-P

But could see if there's some emulator etc.

**Comment by tuomas2** [comment]
BTW, maybe this is something you could propose in #2874
